### ISSUE TITLE
process of getting passes is inaccessible to screen reader users

### ISSUE THREAD

**Issue by venkateshpotluri** [privacypass/challenge-bypass-extension#183]
**Describe the bug**
From the 
>What do I have to do to acquire new passes?

FAQ of the readme on this repository, a user has to get passes by visiting • Click "Get More Passes" in the extension pop-up or navigate to https://captcha.website page, where they have to prove that they are human. Here, the only option seems to be to either click on images or go and create an accessibility signup, both of which are inaccessible and require additional steps to be performed by blind and visually impaired screen reader users to bypass an already inaccessible service, Captcha.

**To Reproduce**
1. go to https://captcha.website.
2. click on I am human checkbox. You will not see an option for an audio captcha or any additional accessible captcha formats there.

**Expected behavior**
There should be options that are accessible to people with disabilities to bypass the challenge. For example, in this case, there is no way for a blind or visually impaired screen reader user to get past this directly on that page, thus preventing them from getting initial passes for the first time, and from getting more passes to be proved as a human on the internet.

**System (please complete the following information):**
 - OS: Windows 10.
 - Cloudflare tokens or hCaptcha tokens? I think hCaptcha.
 - Browser: Chrome
 - Browser Version: 83.0.4103.106
 - Privacy Pass Version: 2.0.6
 - Did you install Privacy Pass from this repository or from the browser store? Chrome web store

**Additional context**
[this document on inaccessibility of captcha](https://www.w3.org/TR/turingtest/#:~:text=CAPTCHAs%20are%2C%20consequently%2C%20allowed%20under,sensory%20perception%20are%20provided%20to) could be of help.

**Comment by alxdavids** [comment]
Hi, thank you for bringing this up and for the detailed report. I'm not an expert on accessibility and so highlighting these concerns for me is really useful.

Am I correct in saying that the problem here is that the CAPTCHAs are themselves inaccessible? In other words, we need Cloudflare and hCaptcha to provide alternative challenges that users would be able to solve and receive Privacy Pass tokens from?

**Comment by venkateshpotluri** [comment]
yes. Cloudflare and hCaptcha should provide alternate captcha formats for users to solve to get passes. Also, users should easily be able to discover these, and the alternate formats should be usable with accessibility technology like screen readers and switch controls.
